CDP4DalCommon/Protocol/Tasks/CometTask.cs

```diff
@@ -71,7 +71,7 @@ public CometTask()
         /// <remarks>
         /// A value of -1 is returned when the task is still running or not completed with success
         /// </remarks>
-        public readonly int Duration => this.ComputeDuration();
+        public readonly int Duration => ComputeDuration();
```

Comment:
please revert and keep using `this.` syntax

Reply:
done